docs/gh_pages/docs/safe_loader.md

```diff
@@ -23,7 +23,7 @@ This page consist of 4 primary tabs that provides the following details:
 2. **Findings**:
    The figure denotes the cumulative count of Topics and Entities identified across all applications. It will also furnish you with a comprehensive list of these Topics and Entities, along with supplementary information including the count of source files they originate from, the Datasource, and the name of the Application.
 
-3. **Files with Findings**:
+3. **Documents with Findings**:
```

Comment:
The title refers to _"Documents"_, while the description mentions _"files"_. Please confirm if the terms are accurate.

Reply:
Made file to Documents everywhere as our UI has documents and not files